### ISSUE TITLE
CONVOLUTIONAL AUTOENCODER (CAE) conv-encode Layer

### ISSUE THREAD

**Issue by glazari** [sjchoi86/advanced-tensorflow#3]
In creating the 3 convolutional encoding layer you accidentally save the dropout of this layer to the wrong name

`    _ce3 = tf.nn.sigmoid(tf.add(tf.nn.conv2d(_ce2, _W['ce3']
        , strides=[1, 2, 2, 1], padding='SAME'), _b['be3'])) 
   
  _ce1 = tf.nn.dropout(_ce3, _keepprob)`  

When it should be

`    _ce3 = tf.nn.sigmoid(tf.add(tf.nn.conv2d(_ce2, _W['ce3']
        , strides=[1, 2, 2, 1], padding='SAME'), _b['be3'])) 
    
_ce3 = tf.nn.dropout(_ce3, _keepprob)`

Probably doesn't affect performance, as it just skips the drop out for one layer. But still, though it would be nice to point this out.
